specification/monitor/resource-manager/Microsoft.Insights/preview/2020-01-01-preview/managementGroupDiagnosticSettings_API.json

```diff
@@ -0,0 +1,342 @@
+        "tags": [
+          "ManagementGroupDiagnosticSettings"
+        ],
+        "operationId": "ManagementGroupDiagnosticSettings_CreateOrUpdate",
+        "description": "Creates or updates management group diagnostic settings for the specified resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ManagementGroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagementGroupDiagnosticSettingsResource"
+            },
+            "description": "Parameters supplied to the operation."
+          },
+          {
+            "$ref": "#/parameters/NameParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to create a management group diagnostic setting",
+            "schema": {
+              "$ref": "#/definitions/ManagementGroupDiagnosticSettingsResource"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Creates or Updates the management group diagnostic setting": {
+            "$ref": "./examples/createOrUpdateManagementGroupDiagnosticSetting.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ManagementGroupDiagnosticSettings"
+        ],
+        "operationId": "ManagementGroupDiagnosticSettings_Delete",
+        "description": "Deletes existing management group diagnostic settings for the specified resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ManagementGroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/NameParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to remove a management group diagnostic setting"
+          },
+          "204": {
+            "description": "Successful request to remove a management group diagnostic setting"
+          }
+        },
+        "x-ms-examples": {
+          "Deletes the management group diagnostic setting": {
+            "$ref": "./examples/deleteManagementGroupDiagnosticSetting.json"
+          }
+        }
+      }
+    },
+    "/providers/microsoft.management/managementGroups/{managementGroupId}/providers/microsoft.insights/diagnosticSettings": {
```

Comment:
this should include "x-ms-pageable": true even if you don't support nextLinks today.  Prevents breaking changes if you add them in the future

Reply:
@pilor  where exactly should this be added?

Reply:
search the repo for the below snippet.  It goes anywhere in the GET collection operation

```
"x-ms-pageable":` {
          "nextLinkName": null
        }
```